packages/ckeditor5-code-block/tests/outdentcodeblockcommand.js

```diff
@@ -132,6 +132,30 @@ describe( 'OutdentCodeBlockCommand', () => {
 			expect( outdentCommand.isEnabled ).to.be.false;
 		} );
 
+		it( 'should not break with inline elements at the end of line', () => {
```

Comment:
I don't like these expects that we have here and the names for these tests. Make these tests analogical to the tests above. They should check if the command is enabled or not. And they should be named accordingly ("it should be false when ..."). Then, I think I am missing tests that show the positive scenario, i.e., there is a leading sequence in a line that includes inline element.